core/lib/RemoteAction.rei

```diff
@@ -0,0 +1,18 @@
+/***
+ * RemoteAction provides a way to send actions to a remote component.
```

Comment:
I'm not familiar with remote actions. What's the difference between them and React context?

Reply:
It seems similiar in our concept to our `Event.t` in `reactify`: <https://github.com/revery-ui/reason-reactify/blob/master/lib/Event.re> , just with multiple subscribers instead of one.